src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -171,8 +172,9 @@ const _sendTransaction = async (
     transactionHash = sendTransactionResult.transactionHash;
     job.log(`Sent transaction: ${transactionHash}`);
   } catch (error: unknown) {
-    // If NonceAlreadyUsedError, rebase the nonce and retry.
-    if (isNonceAlreadyUsedError(error)) {
+    // If NonceAlreadyUsedError, which can also manifest as a ReplacementGasFeeTooLowError,
+    // recycle the nonce and retry the transaction.
+    if (isNonceAlreadyUsedError(error) || isReplacementGasFeeTooLow(error)) {
```

Comment:
We don't need `isReplacementGasFeeTooLow` here.

We can avoid this error totally, if we use the `max(onchain-gas, previous-tx-gas-values)` for gas values of the new transactions.